sink/sqs.go

```diff
@@ -0,0 +1,164 @@
+package sink
+
+import (
+	"strconv"
+	"time"
+
+	"os"
+
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	log "github.com/sirupsen/logrus"
+)
+
+// SQS ...
+type SQSSink struct {
+	session   *session.Session
+	sqs       *sqs.SQS
+	queueName string
+	groupId   string
+	stopCh    chan interface{}
+	putCh     chan []byte
+	batchCh   chan [][]byte
+}
+
+// NNewSQS ...
+func NewSQS(groupId string) (*SQSSink, error) {
+	queueName := os.Getenv("SINK_SQS_QUEUE_URL")
+	if queueName == "" {
+		return nil, fmt.Errorf("[sink/sqs] Missing SINK_SQS_QUEUE_URL")
+	}
+
+	sess := session.Must(session.NewSession())
+	svc := sqs.New(sess)
+
+	return &SQSSink{
+		session:   sess,
+		sqs:       svc,
+		queueName: queueName,
+		groupId:   groupId,
+		stopCh:    make(chan interface{}),
+		putCh:     make(chan []byte, 10000),
```

Comment:
can we keep this at `1000` like the other sinks?

Reply:
sure